package.json

```
@@ -55,35 +55,43 @@
     "commands": [
       {
         "command": "gitguardian.quota",
-        "title": "gitguardian: Show quota"
+        "title": "Show quota",
+        "category": "GitGuardian"
       },
       {
         "command": "gitguardian.ignore",
-        "title": "gitguardian: Ignore last found incidents"
+        "title": "Ignore last found incidents",
+        "category": "GitGuardian"
       },
       {
         "command": "gitguardian.authenticate",
-        "title": "gitguardian: Authenticate"
+        "title": "Authenticate",
+        "category": "GitGuardian"
       },
       {
         "command": "gitguardian.logout",
-        "title": "gitguardian: logout"
+        "title": "logout",
```

Just to be consistent with all titles starting with a capital letter
```suggestion
        "title": "Logout",
```